$NetBSD: patch-cl,v 1.10 2013/12/08 13:08:59 prlw1 Exp $

Leave gdbus-2.0/codegen sudirectory to devel/gdbus-codegen package.
Don't install completion data.
(patch-an)
--- gio/Makefile.am.orig	2013-10-15 19:07:46.000000000 +0000
+++ gio/Makefile.am
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk
 
-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS = 
 
 if OS_UNIX
 SUBDIRS += xdgmime
@@ -718,13 +718,6 @@ gdbus_LDADD = libgio-2.0.la \
 	$(top_builddir)/glib/libglib-2.0.la 		\
 	$(top_builddir)/gobject/libgobject-2.0.la
 
-completiondir = $(datadir)/bash-completion/completions
-completion_DATA = \
-	completion/gdbus				\
-	completion/gsettings				\
-	completion/gresource
-EXTRA_DIST += $(completion_DATA)
-
 # ------------------------------------------------------------------------
 # gresource tool
 
